src/v2/Apps/Artwork/Components/ArtworkDetails/__tests__/ArtworkDetails.jest.tsx

```diff
@@ -217,7 +217,7 @@ describe("ArtworkDetails", () => {
           type: "NOT Auction House",
         },
       })
-      expect(wrapper.html()).toContain("Following")
+      expect(wrapper.html()).toContain("Follow")
```

Comment:
Are these changes meant to match different button states, or is the label actually "Follow?"

Reply:
I just looked at the artwork page in an attempt to answer my own question and saw multiple "Follow" buttons. If it's important to this test, might be worth scoping the assertion.

Reply:
I updated the test fixture to be unfollowed by default, so this changed a few tests that look for text matching. Shouldn't affect any functionality.